kbase-extension/static/kbase/js/widgets/function_output/kbaseReadsSetView.js

```diff
@@ -50,20 +50,26 @@ define (
                     $self.link_ref = $self.obj_ref;
                 }
 
-                $self.ws = new Workspace(Config.url('workspace'),
-                                         {'token':$self.authToken()});
-                
-                return Promise.resolve($self.ws.get_objects([{'ref': this.obj_ref}]))
-                    .then(function (data) {
-                        var info = data[0]["info"],
+                this.setAPI = new SetAPI(Config.url('service_wizard'),
+                                         {'token': this.authToken()},
+                                         "dev");
+                this.wsClient = new Workspace(Config.url('workspace'),
```

Comment:
We should switch away from using the WS to get item information- the SetAPI can give you ws metadata for each item.  If there is something we want to show in the set viewer for each item, we should add it to the automatic metadata so we don't pull all the reads ws data into the client (looks like you aren't using subdata either, but just pulling in all the reads data- they are small objects but there could be a lot of them in a set...).